src/openrct2/drawing/Font.cpp

```diff
@@ -74,325 +74,307 @@ int32_t font_sprite_get_codepoint_offset(int32_t codepoint)
 {
     switch (codepoint)
     {
-        case FORMAT_ENDQUOTES:
+        case UnicodeChar::quote_close:
```

Comment:
Can I suggest we replace this enormous switch block with a map combined with a binary search? Right now, the lookup runs in O(n), but with a binary search algorithm it should run in O(lg n).